zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -261,6 +262,18 @@ def build_customprofile_field(customprofile_field: List[ZerverFieldsT], fields:
             customprofile_field.append(customprofilefield)
     return custom_field_map, customprofilefield_id
 
+def process_slack_custom_fields(user: ZerverFieldsT,
+                                slack_user_custom_field_map: ZerverFieldsT) -> None:
+    slack_user_custom_field_map[user['id']] = {}
+    if user['profile'].get('fields'):
+        # Make sure the content of fields is not 'None'
```

Comment:
Minor: Maybe remove or move the comment, since the check is the clause of the conditional?

I was expecting `if 'fields' in user['profile'] and user['profile']['fields]`, but this seems to work well, if not better :)

Reply:
Removed the comment.